public/controllers/register-agent/utils/validations.tsx

```diff
@@ -0,0 +1,51 @@
+export const ValidateServerAddress = value => {
```

Comment:
**nitpick**: use camelCase for the constant

Reply:
Resolved in commit [224b1e4](https://github.com/wazuh/wazuh-kibana-app/pull/5554/commits/224b1e4ffff4a917a5159333b7b481961c422017)